spec/Powertrain/PowerSource/ElectricMotor.vspec

```diff
@@ -10,3 +11,99 @@
 #
 
 
+- MaxPower:
+  datatype:  uint16
+  type: attribute
+  default: 0
+  unit: kW
+  description: Peak power, in kilowatts, that motor(s) can generate.
+
+- MaxTorque:
+  datatype:  uint16
+  type: attribute
+  default: 0
+  unit: N.m
```

Comment:
N.m or Nm?

Reply:
vss-tools defines "N.m", but that seems wrong.  Let's open a ticket to fix it.

Reply:
Clean up later: https://github.com/GENIVI/vehicle_signal_specification/issues/173